packages/@sanity/components/src/previews/DefaultPreview.js

```diff
@@ -54,13 +47,16 @@ class DefaultPreview extends React.PureComponent {
 
     if (isPlaceholder) {
       return (
-        <div className={media ? styles.hasMedia : ''}>
-          {PLACEHOLDER}
+        <div className={styles.placeholder}>
+          <div className={styles.media} />
+          <div className={styles.heading}>
+            <h2 className={styles.title}>Loading…</h2>
+            <h3 className={styles.subtitle}>Loading…</h3>
```

Comment:
Should it be `Loading…` on both `title` and `subtitle`?

Reply:
This text is there only to make the placeholder render correctly. It is transparent and not visible.